carbonserver/carbonserver/api/routers/projects.py

```diff
@@ -16,7 +16,7 @@
 projects_temp_db = []
 
 
-@router.put("/projects", tags=PROJECTS_ROUTER_TAGS, status_code=status.HTTP_201_CREATED)
+@router.put("/project", tags=PROJECTS_ROUTER_TAGS, status_code=status.HTTP_201_CREATED)
```

Comment:
Could we suffix with / all routers by default ?

Reply:
Trailing / removed to conform with "Rule #1: A trailing forward slash (/) should not be included in URIs" https://blog.restcase.com/7-rules-for-rest-api-uri-design/